clang/lib/Driver/ToolChains/WebAssembly.cpp

```diff
@@ -61,6 +61,10 @@ std::string wasm::Linker::getLinkerPath(const ArgList &Args) const {
   return ToolChain.GetProgramPath(ToolChain.getDefaultLinker());
 }
 
+static bool IsWasip2(const llvm::Triple &TargetTriple) {
+  return TargetTriple.getOSName() == "wasip2";
```

Comment:
I guess WASI preview3 and the future will be the same. Should we invert the test and only check for `wasip1` ?

Reply:
Sounds reasonable to me yeah 👍